config/eu_cookie_compliance.settings.json

```diff
@@ -1,34 +1,71 @@
 {
-  "_config_name": "eu_cookie_compliance.settings",
-  "eu_cookie_compliance_domain": "",
-  "eu_cookie_compliance_cookie_lifetime": 100,
-  "eu_cookie_compliance_en": {
-    "popup_enabled": 1,
-    "popup_clicking_confirmation": 1,
-    "popup_position": 1,
-    "popup_agree_button_message": "OK, I agree",
-    "popup_disagree_button_message": "No, give me more info",
-    "popup_info": {
-	  "value": "<h2>We use cookies on this site to enhance your user experience</h2> <p>By clicking any link on this page you are giving your consent for us to set cookies.</p>",
-      "format": "full_html"
-	  },
-    "popup_agreed_enabled": 1,
-    "popup_hide_agreed": 0,
-    "popup_find_more_button_message": "More info",
-    "popup_hide_button_message": "Hide",
-    "popup_agreed": {
-      "value": "<h2>Thank you for accepting cookies</h2> <p>You can now hide this message or find out more about cookies.</p>",
-      "format": "full_html"
+    "_config_name": "eu_cookie_compliance.settings",
+    "_module": "eu_cookie_compliance",
+    "eu_cookie_compliance_domain": "",
+    "eu_cookie_compliance_cookie_lifetime": "100",
+    "eu_cookie_compliance": {
```

Comment:
It seems that this has previously been `eu_cookie_compliance_en` so an update hook might be needed.

Reply:
...yep. https://backdropcms.org/project/usage/eu-cookie-compliance shows about a dozen sites using it.

Reply:
yes, added one to bring the settings back in line with the 1.x branch `eu_cookie_compliance_update_1001()`.